.lando.yml

```diff
@@ -187,6 +187,17 @@ services:
       environment:
         ELASTICVUE_CLUSTERS: '[{"name": "dev cluster", "uri": "http://localhost:9200", "password": "", "user": ""}]'
 
+  # Uncomment this section if you prefer to use the mode advanced Kibana tool with Elasticsearch:
+  #
+  # kibana:
+  #   type: lando
+  #   services:
+  #     image: "docker.elastic.co/kibana/kibana:7.17.0"
+  #     command: "/bin/tini -- /usr/local/bin/kibana-docker"
+  #     user: kibana
+  #     ports:
+  #       - "5601:5601"
+
```

Comment:
Some people might still want to use kibana, so we can add it back as a commented out service (FYI @misterjoonas )

Reply:
An idea: You could put this in a `lando.local.yml.example` file; if someone wants kibana they can rename it to `.lando.local.yml` without ending up with a .lando.yml file that isn't in sync with the repo

Reply:
hi @badrange ! 

the thing is that this is not really meant to be updated after you have used it once to kickstart your project

Reply:
Ah, right, makes sense!